diff -uNr asc-1.10.2.orig/configure.in asc-1.10.2/configure.in
--- asc-1.10.2.orig/configure.in	Sat Apr 27 18:32:43 2002
+++ asc-1.10.2/configure.in	Sat Jul 27 11:15:08 2002
@@ -299,11 +299,6 @@
 
 
 
-dnl for libtool to work LTLIBOJS is needed (currently bug of autoconf)
-Xsed="sed -e s/^X//"
-LTLIBOBJS=`echo X"$LIBOBJS"|[$Xsed -e "s,\.[^.]* ,.lo ,g;s,\.[^.]*$,.lo,"]`
-AC_SUBST(LTLIBOBJS)
-
 AC_OUTPUT( Makefile source/Makefile source/docs/Makefile \
 	   source/docs/en/Makefile source/sdl/Makefile \
 	   source/libs/triangul/Makefile source/libs/Makefile \
