diff -durN asc-1.9.4.orig/configure.in asc-1.9.4/configure.in
--- asc-1.9.4.orig/configure.in	Thu Oct 11 12:28:56 2001
+++ asc-1.9.4/configure.in	Fri Oct 12 20:42:49 2001
@@ -56,7 +56,7 @@
 AC_TYPE_SIZE_T
 
 dnl Checks for libraries.
-AC_CHECK_LIB( pthread, pthread_create, [ ] )
+AC_CHECK_LIB( pthread, pthread_create, )
 AC_CHECK_LIB( SDL_image, IMG_LoadJPG_RW,,  AC_MSG_ERROR( ASC requires the SDL_image library. You can get it at www.libsdl.org ) )
 AC_CHECK_LIB( SDL_mixer, Mix_PlayChannelTimed,, AC_MSG_ERROR( ASC requires the SDL_mixer library. You can get it at www.libsdl.org ) )
 AC_LANG_SAVE
@@ -69,7 +69,7 @@
 AC_ARG_ENABLE( libbz2-static, [  --enable-libbz2-static  compiles a static \
 libbz2 into the binary of asc.], [lib_bz2_static=$enableval], \
 [lib_bz2_usecache=yes] )
-AC_CHECK_LIB( bz2, BZ2_bzReadOpen, [ ], [checklib_bz2_static=yes] )
+AC_CHECK_LIB( bz2, BZ2_bzReadOpen, , [checklib_bz2_static=yes] )
 AC_CACHE_CHECK( [for static libbz2 flag state], [ac_cv_lib_bz2_static],
 [ ac_cv_lib_bz2_static=$lib_bz2_static ] )
 dnl ac_cv_lib_bz2_static holds cached flag state, checklib_bz2... holds check
@@ -150,10 +150,11 @@
 dnl what gehts packed into main.con (the data container file)
 AC_SUBST([MAINCONDATA])
 
-AC_OUTPUT(Makefile source/Makefile source/docs/Makefile source/docs/en/Makefile source/sdl/Makefile source/libs/triangul/Makefile source/libs/Makefile source/ai/Makefile source/unix/Makefile )
+AC_OUTPUT(Makefile source/Makefile source/docs/Makefile source/docs/en/Makefile source/sdl/Makefile source/libs/triangul/Makefile source/libs/Makefile source/ai/Makefile source/unix/Makefile \
 doc/manpages/Makefile \
 source/Makefile source/docs/Makefile source/clparser/Makefile \
 source/docs/en/Makefile source/sdl/Makefile source/libs/triangul/Makefile \
 source/libs/Makefile source/ai/Makefile source/unix/Makefile \
-source/unix/mapeditor/Makefile source/tools/Makefile source/tools/linux/Makefile)
+source/unix/mapeditor/Makefile source/tools/Makefile source/tools/linux/Makefile \
+data/Makefile data/moredata/Makefile doc/Makefile )
 
